README.md

```diff
@@ -1,34 +1,42 @@
 # OCaml 5 Tutorial
 
 A hands-on tutorial on the new parallelism features in OCaml 5. This tutorial
-was run on 19th May 2022.
+was run on 5 July 2022. Currently, the alpha version of OCaml 5 has been released, and the full version is set for release in September 2022. 
 
 ## Installation
 
-This tutorial works on x86-64 and Arm64 architectures on Linux and macOS.
+This tutorial works on x86-64 and Arm64 architectures on Linux and macOS. 
 
-With `opam` version >= 2.1:
+Before we move on to the instructions, check your version of opam with `opam --version`, then follow the instructions below for your version. You can also quickly update to the latest version of opam (currently 2.1.2) by running:
+
+```
+bash -c "sh <(curl -fsSL https://raw.githubusercontent.com/ocaml/opam/master/shell/install.sh)"
+```
+
+With opam version >= 2.1:
 
 ```bash
 opam update
-opam switch create 5.0.0+trunk --repo=default,alpha=git+https://github.com/kit-ty-kate/opam-alpha-repository.git
+opam switch create 5.0.0~alpha0 --repo=default,alpha=git+https://github.com/kit-ty-kate/opam-alpha-repository.git
 opam install . --deps-only
+eval $(opam env)
 ```
 
-with `opam` version < 2.1:
+with opam version < 2.1:
 
 ```bash
 opam update
-opam switch create 5.0.0+trunk --repo=default,beta=git+https://github.com/ocaml/ocaml-beta-repository.git,alpha=git+https://github.com/kit-ty-kate/opam-alpha-repository.git
+opam switch create 5.0.0~alpha0 --repo=default,beta=git+https://github.com/ocaml/ocaml-beta-repository.git,alpha=git+https://github.com/kit-ty-kate/opam-alpha-repository.git
 opam install . --deps-only
+eval $(opam env)
 ```
 
 Since we will be doing performance measurements, it is recommended that you also
-install [`hyperfine`](https://github.com/sharkdp/hyperfine).
+install [`hyperfine`](https://github.com/sharkdp/hyperfine). With Homebrew, `brew install hyperfine`
```

Comment:
Homebrew is macOS only. Is there a specific reason only to include homebrew instructions here? `hyperfine` README includes installation instructions for a variety of platforms. I would suggest that we drop this addition.

Reply:
Sure, let's just drop it.